treq/client.py

```diff
@@ -132,7 +133,7 @@ def request(self, method, url, **kwargs):
             url = _combine_query_params(url, params)
 
         if isinstance(url, unicode):
-            url = url.encode('ascii')
+            url = URL.fromText(url).asURI().asText().encode('us-ascii')
```

Comment:
There is none.  I mistakenly though that `us-ascii` was the canonical form, but https://docs.python.org/2/library/codecs.html#standard-encodings reveals that to be false.  I will correct this.